src/util.ts

```diff
@@ -502,21 +502,21 @@ export function timeout(millis): Promise<void> {
 }
 
 /**
- * Exapnds ~ to homedir in non-Windows platform and resolves ${workspaceRoot}
+ * Exapnds ~ to homedir in non-Windows platform and resolves ${workspaceFolder}
  */
-export function resolvePath(inputPath: string, workspaceRoot?: string): string {
+export function resolvePath(inputPath: string, workspaceFolder?: string): string {
```

Comment:
The `resolvePath` function was already supporting both `$workspaceRoot` and `$workspaceFolder`. We want to retain that. So, please revert the changes done to `resolvePath` function